mmeds/util.py

```diff
@@ -182,6 +182,58 @@ def join_metadata(subject, specimen, subject_type):
     return df
 
 
+def split_metadata(full, clean_for_meta_analysis=True, id_col=('RawData', 'RawDataID'), new_study_name=None):
+    """ Splits a full metadata file into two dataframes, subject and specimen """
```

Comment:
large method, consider if the code could be slightly better organized. Can't find something entirely wrong but would recommend taking a critical eye to this, consider how it may be used in the future

Reply:
reasonable, leave a TODO maybe linking an issue